pkg/discovery/network/ovnkubernetes.go

```diff
@@ -34,8 +34,9 @@ const (
 )
 
 func discoverOvnKubernetesNetwork(clientSet kubernetes.Interface) (*ClusterNetwork, error) {
-	ovnDBPod, err := findPod(clientSet, "name=ovnkube-db")
+	ovnDBPods, err := FindPod(clientSet, "name=ovnkube-db")
 
+	ovnDBPod := &ovnDBPods.Items[0]
 	if err != nil || ovnDBPod == nil {
```

Comment:
This is will panic if no pod is found, ie list is empty.

```suggestion
	if err != nil || len(ovnDBPods.Items) == 0 {
```